scripts/mkcloud

```diff
@@ -675,6 +675,11 @@ function restartcloud()
 # bring up VMs that will take cloud controller/compute/storage roles
 function setupnodes()
 {
+    local nodenumbercontroller=1
+    if [[ $clusterconfig == *"services"* ]]; then
```

Comment:
right, but the rest of qa_crowbarsetup.sh puts all services on a cluster called services.. so it is not only freeform :/